lib/improver/tests/nbhood/test_neighbourhoodprocessing.py

```diff
@@ -153,6 +152,60 @@ def set_up_cube(zero_point_indices=((0, 0, 7, 7),), num_time_points=1,
     return cube
 
 
+def set_up_cube_with_no_realizations(zero_point_indices=((0, 7, 7),),
+                                     num_time_points=1,
+                                     num_grid_points=16,
+                                     source_realizations=None):
```

Comment:
There is a lot of code duplicated from the set_up_cube method. Can these be refactored? Or does the extra dimension make this impractical?